app/Controllers/Talk.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Controllers;
+
+use App\Controllers\BaseController;
+use App\Models\TalkModel;
+use App\Models\EventModel;
+
+class Talk extends BaseController
+{
+    public function getICS($year) {
```

Comment:
Since this endpoint collects all the talks for a given event, wouldn't it be clever to just move the code into the `Event` controller?
The route could be like this:
```php
$routes->get('events/(:num)/ics', [Event::class, 'getICS']);
```